jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -653,15 +653,9 @@ void java_bytecode_parsert::rconstant_pool()
 
   constant_pool.resize(constant_pool_count);
 
-  for(constant_poolt::iterator
-      it=constant_pool.begin();
-      it!=constant_pool.end();
+  for(auto it = std::next(constant_pool.begin()); it != constant_pool.end();
```

Comment:
Could the constant pool be empty? If so this is wrong

Reply:
No, this is already checked at the start of the function.